hawkular-inventory-itest-parent/hawkular-inventory-itest-dist/pom.xml

```diff
@@ -0,0 +1,592 @@
+                          <value>${uuid.hawkular.accounts.backend}</value>
+                        </parameter>
+                      </parameters>
+                    </transformationSet>
+                  </transformationSets>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <configuration>
+              <finalName>${project.artifactId}-${project.version}</finalName>
+              <appendAssemblyId>false</appendAssemblyId>
+              <descriptors>
+                <descriptor>assembly.xml</descriptor>
+              </descriptors>
+            </configuration>
+            <executions>
+              <execution>
+                <id>distro-assembly</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>single</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+
+          <plugin>
+            <groupId>com.spotify</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>0.2.12</version>
+            <configuration>
+              <imageName>hawkular:snapshot</imageName>
+              <dockerDirectory>docker</dockerDirectory>
+              <resources>
+                <resource>
+                  <targetPath>/</targetPath>
+                  <directory>${project.build.directory}</directory>
+                  <include>hawkular-${project.version}.zip</include>
+                </resource>
+              </resources>
+            </configuration>
+          </plugin>
+
+          <!-- Generate some keys for Keycloak integration, so that we don't ship with default credentials -->
+          <plugin>
+            <groupId>org.codehaus.gmaven</groupId>
+            <artifactId>gmaven-plugin</artifactId>
+            <version>1.5</version>
+            <configuration>
+              <providerSelection>1.8</providerSelection>
+            </configuration>
+            <executions>
+              <execution>
+                <id>set-custom-property</id>
+                <phase>initialize</phase>
+                <goals>
+                  <goal>execute</goal>
+                </goals>
+                <configuration>
+                  <source>
+                    import java.util.UUID
+                    def uuidBackend = UUID.randomUUID().toString()
+                    def uuidUi = UUID.randomUUID().toString()
+                    project.properties.setProperty('uuid.hawkular.accounts.backend', uuidBackend.toString())
+                    project.properties.setProperty('uuid.hawkular.ui', uuidUi.toString())
+                  </source>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <profile>
+      <id>run-test-dist-and-tests</id>
```

Comment:
Why do we need to keep `run-test-dist-and-tests` profile separate from `compose-test-dist` ?